src/serac/numerics/functional/element_restriction.cpp

```diff
@@ -267,17 +271,29 @@ axom::Array<DoF, 2, axom::MemorySpace::Host> GetElementRestriction(const mfem::F
   }
 
   if (n == 0) {
+#ifdef SERAC_USE_RAJA
```

Comment:
I think this should be `SERAC_USE_UMPIRE`

Reply:
Yeah makes sense, basically all of the `axom::MemorySpace::Host` stuff needs to be fixed.